test/Integration/hoist-vector-transfer-brgemm.mlir

```diff
@@ -3,38 +3,30 @@
 // RUN: diff %t.1 %t.2 | FileCheck %s --check-prefix=DIFF --allow-empty
 
 // DIFF-NOT: {{.}}
```

Comment:
You don't need this. You can just run `diff` and if there's anything it returns non-zero and fails the test.

Reply:
Okay. Removed the FileCheck.